sources/zh/piaotian.py

```diff
@@ -2,6 +2,28 @@
 import logging
 
 from lncrawl.core.crawler import Crawler
+import urllib.parse
+
+headers = {
```

Comment:
does it not work without these headers? the cloudscraper automatically set headers using rotating user agents. if we have these headers, that feature will not be used.

Reply:
Is removing the `User-Agent` header alone enough?

Reply:
Is removing `User-Agent` enough for the feature to work?